### PR TITLE
Adds burr version of log app

### PR DESCRIPTION
**Pull request by skrawcz**
Notes:
 - llama index prompt doesn't need convo history

otherwise WIP branch for adding things

### CODE REVIEW

**Comment by skrawcz** [comment]
to run, install "burr[start,opentelemetry]".

Then in separate terminal run `burr` to get the burr UI up.

**Comment by hugobowne** [comment]
thanks @skrawcz . can you add any dependencies to `pyproject.toml` and confirm that it works in codespaces?

File: pyproject.toml [hunk 1]
```diff
@@ -13,7 +13,9 @@ pymupdf = "^1.24.10"
 llama-index-llms-ollama = "^0.3.2"
 sentence-transformers = "^3.1.1"
 llama-index-embeddings-huggingface = "^0.3.1"
-
+# add burr[start,opentelemetry] to the dependencies
+torch = "2.2.2"
```
Comment (skrawcz) [inline pyproject.toml:17]: this is for my mac... I can't install 2.5 for my mac.

**Comment by skrawcz** [comment]
> thanks @skrawcz . can you add any dependencies to `pyproject.toml` and confirm that it works in codespaces?

I think I got it working?

**Pull request closed without merging**